README.md

```diff
@@ -102,3 +102,12 @@ straightforward using these steps:
       route `/api/kernels` is not enabled by default for greater security. See
       the `--KernelGatewayApp.list_kernels` parameter documentation if you
       would like to enable the `/api/kernels` route.)
+
+### Swagger Documentation
```

Comment:
@parente, here are the docs for updating the swagger specs. Let me know if you would like more.